src/units.rs

```diff
@@ -605,6 +637,20 @@ pub fn divide(left: Number, right: Number) -> Result<Number, String> {
     let hours = convert(right, Hour)?;
     let kph = Number::new(kilometers.value / hours.value, KilometersPerHour);
     Ok(convert(kph, final_unit)?)
+  } else if lcat == Length && rcat == Speed {
+    // 12 km / 100 kph
+    let final_unit = match (left.unit, right.unit) {
+      (Kilometer, KilometersPerHour) => Hour,
+      (Meter, MetersPerSecond) => Second,
+      (Mile, MilesPerHour) => Hour,
+      (Foot, FeetPerSecond) => Second,
+      (NauticalMile, Knot) => Hour,
+      _ => Hour,
+    };
+    let kilometers = convert(left, Kilometer)?;
+    let kilometers_per_hour = convert(right, KilometersPerHour)?;
+    let hour = Number::new(kilometers.value / kilometers_per_hour.value, Hour);
```

Comment:
Now that `to_ideal_unit()` always decides a `Time` unit, this match statement doesn't do anything. Should work to do this:
```rust
let kilometers = convert(left, Kilometer)?;
let kilometers_per_hour = convert(right, KilometersPerHour)?;
let hour = Number::new(kilometers.value / kilometers_per_hour.value, Hour);
Ok(to_ideal_unit(hour))
```